core/src/main/java/com/linecorp/armeria/common/ContentDisposition.java

```diff
@@ -163,9 +162,10 @@ public static ContentDisposition parse(String contentDisposition) throws Unsuppo
                     final int idx1 = value.indexOf('\'');
                     final int idx2 = value.indexOf('\'', idx1 + 1);
                     if (idx1 != -1 && idx2 != -1) {
-                        charset = Charset.forName(value.substring(0, idx1).trim());
-                        checkArgument(UTF_8.equals(charset) || ISO_8859_1.equals(charset),
-                                      "Charset: %s (expected: UTF-8 or ISO-8859-1)", charset);
+                        final String charsetString = value.substring(0, idx1).trim();
+                        charset = supportedCharsets.get(Ascii.toLowerCase(charsetString));
+                        checkArgument(charset != null,
+                                      "Charset: %s (expected: UTF-8 or ISO-8859-1)", charsetString);
```

Comment:
Didn't we decide to fall back to iso-8859-1?

Reply:
I missed adding newly changed files to staged one by `git add .`  😅
https://github.com/line/armeria/pull/3253/commits/4eb127c8e81a552399ac759998a05ec0ea4a48c2